src/OpenTelemetry.Contrib.Instrumentation.Remoting/README.md

```diff
@@ -0,0 +1,124 @@
+# .NET Remoting Instrumentation for OpenTelemetry.Contrib .NET
+
+This is an instrumentation library, which instruments [.NET Remoting](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.0/72x4h507(v=vs.85))
+and collects telemetry about incoming and outgoing requests on client
+and server objects.
+
+.NET Remoting is a [legacy technology](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.0/kwdt6w2k(v=vs.85))
+that shouldn't be used for new .NET applications and [doesn't exist](https://docs.microsoft.com/dotnet/core/porting/net-framework-tech-unavailable#remoting)
+in .NET Core at all. However, if you do have a legacy application you are
+looking to instrument, consider using this package.
+
+## Installation
+
+```shell
+dotnet add package OpenTelemetry.Contrib.Instrumentation.Remoting
+```
+
+## Configuration
+
+To enable .NET remoting instrumentation, call `AddRemotingInstrumentaion()` on
+the `TracerProviderBuilder` during the application startup in both client and
+server code.
+
+For example, in a client console app:
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+
+namespace ExampleClient
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Sdk.CreateTracerProviderBuilder()
```

Comment:
its good to show examples disposing the provider when done. Consider doing 'using' statement.

Reply:
Done